packages/api/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import * as uma from "@uma/sdk";
```

Comment:
Is this file exporting types...?

Reply:
hmm this file probably isnt needed, looks like a mistake

Reply:
ok took a look at this again. its not a mistake, im importing everything but just using the types currently. will change it to show types being imported.